### PR TITLE
ExceptionMap - add new code for missing promotional coupon

### PR DESCRIPTION
**Pull request by cjohannsen**
Related IO Part: https://github.com/plentymarkets/plugin-io/pull/650

### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been tested by the author
- [x] Changes have been tested by the reviewer
- [x] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

File: resources/lang/de/Template.properties [hunk 1]
```diff
@@ -180,9 +180,10 @@ couponOnlyForExistingCustomers = "Der Gutschein kann nur von Bestandskunden eing
 couponOnlyForNewCustomers = "Der Gutschein kann nur von Neukunden eingelöst werden."
 couponOnlySingleUsage = "Diesen Gutschein kann nicht für Abo-Artikel eingelöst werden."
 couponOnlySubscription = "Der Gutschein kann nur für Abo-Artikel eingelöst werden."
+couponPromotionRequired = "Die Variant des Artikels kann nur gekauft werden, wenn der Code eines Aktionsgutscheins eingegeben wird."
```
Comment (cjohannsen) [inline resources/lang/de/Template.properties]: @fmutschler 
Bildschirmfoto 2019-10-29 um 16 46 48

File: resources/lang/en/Template.properties [hunk 2]
```diff
@@ -180,10 +180,11 @@ couponOnlyForExistingCustomers = "The coupon can only be redeemed by existing cu
 couponOnlyForNewCustomers = "The coupon can only be redeemed by new customers."
 couponOnlySingleUsage = "The coupon cannot be redeemed for subscription items."
 couponOnlySubscription = "The coupon can only be redeemed for subscription items."
+couponPromotionRequired = "Customers can only buy the variations of this item if they enter the code of a promotional coupon."
```
Comment (cjohannsen) [inline resources/lang/en/Template.properties]: @fmutschler 
Bildschirmfoto 2019-10-29 um 16 46 58

**Review by felixdausch — APPROVED** [review]
Übersetzungen bitte noch anpassen.

**Comment by fmutschler** [comment]
Übersetzungen angepasst.

**Review by stentrop — APPROVED** [review]

**Comment by maxhelligeplenty** [comment]
Hallo  ,
stentrop approved this pull request.





-- 
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/plugin-ceres/pull/1591#pullrequestreview-311048031

Test.test.test

**Comment by maxhelligeplenty** [comment]
Hallo  ,
Merged #1591 into beta.

-- 
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/plugin-ceres/pull/1591#event-2767739646

Test.test.test

**Pull request merged**